src/trace.js

```diff
@@ -370,15 +376,15 @@ export default {
     }
 
     // set axis scale
-    chart.options.scales.x.min = start;
-    chart.options.scales.x.max = end;
+    chart.options.scales[xAxisName].min = start;
+    chart.options.scales[xAxisName].max = end;
```

Comment:
Consider that if "xAxisName" is "xAxes" then it's an array, not an single object

Reply:
This is not true, all scales in chart.js V3 are single objects